src/common.js

```diff
@@ -397,3 +410,17 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+exports.getProcessEnv = function () {
+  var env = objectAssign({}, process.env);
+
+  if (config.nodepath) {
+    env[PATH] += (process.platform === 'win32' ? ';' : ':') + path.join(process.cwd(), 'node_modules', '.bin') + (process.platform === 'win32' ? ';' : ':');
+  }
+
+  return env;
+};
+
+exports.getPathFromEnv = function (env) {
```

Comment:
Yeah, I saw that. I think that wasn't actually necessary for us to do. We should probably convert everything to `process.env.PATH` for the sake of consistency.

Plus, we would run into issues if someone on unix ever did `export path=foo` (totally valid for a user to do, but it would provide incorrect results for `which()`).